spec/features/self_service/firms_index_spec.rb

```diff
@@ -105,6 +122,26 @@ def and_the_trading_names_section_is_showing_a_prompt_to_add_a_trading_name
     expect(firms_index_page).not_to have_trading_names
   end
 
+  def when_i_delete_the_first_trading_name
+    firms_index_page.trading_names.first.delete_button.click
+  end
+
+  def and_i_can_see_a_success_message
+    expected_message = "#{@first_trading_name_registered_name} has been successfully removed."
```

Comment:
My opinion is we should be using `I18n.t!(...)` consistently in the tests. Seeing as we've decided to localise all the strings, not using them in the tests creates a mixed approach which means every time the wording of a string is tweaked it potentially breaks a test too.